$NetBSD: patch-etc_Makefile.am,v 1.1.1.1 2011/05/23 13:37:52 makoto Exp $

(1) To merge sybmol.rev (if configured with --enable-symbol-rev)
(2) To sort bushu.exand by emacs-internal-code order

--- etc/Makefile.am.orig	2011-04-24 20:50:27.000000000 +0900
+++ etc/Makefile.am	2011-04-25 00:25:25.000000000 +0900
@@ -4,11 +4,22 @@
 
 pkgdata_DATA = $(DISTDATA) bushu.expand bushu.index2
 EXTRA_DIST = $(DISTDATA) mk-bdic
+BATCH_SORT	= ../BATCH-SORT
+RM		= ${RM}
 
-bushu.expand bushu.index2: bushu.rev
-	@echo '*** compiling bushu dictionary ***'
-	@echo '*** this may take a few minutes ***'
+bushu.expand bushu.index2: bushu.rev $(BATCH_SORT)
+	@echo ' *** Taking care bushu.rev and sorting it'
+	@if [ .${SYMBOLREV} = ."yes" ]; then \
+	cat bushu.rev symbol.rev > bushu.rev.tmp ;\
+	echo ' *** symbol.rev will be included in bushu.rev ***'; \
+	else \
+	mv bushu.rev bushu.rev.tmp ;\
+	echo ' *** symbol.rev will NOT be included in bushu.rev ***'; \
+	fi
+	$(EMACS) -q -no-site-file -batch -l $(BATCH_SORT) -f batch-sort-file bushu.rev.tmp bushu.rev
+	@echo ' *** compiling bushu dictionary ***'
+	@echo ' *** this may take a few minutes ***'
 	$(EMACS) -q -no-site-file -batch -l $(srcdir)/mk-bdic
 
 clean:
-	$(RM) bushu.expand bushu.index2
+	$(RM) -f bushu.expand bushu.index2
